kv/txn_test.go

```diff
@@ -201,9 +199,7 @@ func verifyUncertainty(concurrency int, maxOffset time.Duration, t *testing.T) {
 			// hasn't changed (i=0). The logical component will change
 			// internally in a way we can't track, but we want to be just
 			// ahead.if
```

Comment:
Yeah. I can remove the "if" at the end but I'm not sure what the comment as a whole is trying to say.